Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1316,17 +1316,17 @@ CSSScrollAnchoringEnabled:
 
 CSSScrollbarColorEnabled:
   type: bool
-  status: testable
+  status: stable
   category: css
   humanReadableName: "CSS scrollbar-color property"
   humanReadableDescription: "Enable scrollbar-color CSS property"
   defaultValue:
     WebKitLegacy:
       default: false
     WebKit:
-      default: false
+      default: true
     WebCore:
-      default: false
```

Comment:
WebCore would be `false` by default. The value would propagate from WebKit.

Reply:
Fixed.